packages/vite-plugin-cloudflare/tsup.config.ts

```diff
@@ -18,4 +18,11 @@ export default defineConfig([
 		noExternal: ['vite/module-runner'],
 		tsconfig: 'tsconfig.runner.json',
 	},
+	{
+		entry: ['src/workerd-custom-import.cts'],
+		outDir: 'dist',
```

Comment:
Let's move this inside the runner folder as that's where it is used:

```suggestion
		outDir: 'dist/runner',
```

Reply:
good point, done 👍